### PR TITLE
Update PCI-DSS profile for RHEL

### PR DESCRIPTION
**Pull request by marcusburghardt**
#### Description:

PCI-DSS profiles for RHEL products were based on PCI-DSSv3.2.1, which will be retired in March 2024.
There was already a control file for PCI-DSSv4, which was largely reviewed and updated by https://github.com/ComplianceAsCode/content/pull/11214

In this meantime it was also enabled CI tests for PCI-DSS profile by https://github.com/ComplianceAsCode/content/pull/11257

This PR is finally updating the RHEL profiles to consume the PCI-DSSv4 control file.

#### Rationale:

- Keep updated with PCI-DSS.

#### Review Hints:

Specially the test-farm tests should be very important to detect issues
Additional tests can be executed locally with automatus.

### CODE REVIEW

**Comment by marcusburghardt** [comment]
There are already some few issues caught by CI tests. I will fix them before moving this PR from Draft to Ready.

**Comment by marcusburghardt** [comment]
@teacup-on-rockingchair and @dodys , after the third commit in this PR I included "prodtype" in some rules used by SUSE and Ubuntu. Could you take a look if you are fine with these prodtype changes, please?

File: linux_os/guide/system/auditing/package_audit-audispd-plugins_installed/rule.yml [hunk 1]
```diff
@@ -1,5 +1,7 @@
 documentation_complete: true
 
+prodtype: fedora,ol9,rhel9,sle12,sle15,ubuntu1604,ubuntu1804,ubuntu2004
```
Comment (dodys) [inline linux_os/guide/system/auditing/package_audit-audispd-plugins_installed/rule.yml:3]: we currently have this rule only in ubuntu2004

**Review by dodys — COMMENTED** [review]
just a minor comment

File: linux_os/guide/system/auditing/package_audit-audispd-plugins_installed/rule.yml (hunk 1 above)
Comment (marcusburghardt) [reply linux_os/guide/system/auditing/package_audit-audispd-plugins_installed/rule.yml:3]: Thanks. I will remove the previous versions.

Comment (Mab879) [reply linux_os/guide/system/auditing/package_audit-audispd-plugins_installed/rule.yml:3]: We will need to figure out something for RHEL 9 as the package doesn't exist there.

Comment (marcusburghardt) [reply linux_os/guide/system/auditing/package_audit-audispd-plugins_installed/rule.yml:3]: Indeed, the package doesn't exist there. But the rule was already already in the data stream by the absence of `prodtype`.

Comment (vojtapolasek) [reply linux_os/guide/system/auditing/package_audit-audispd-plugins_installed/rule.yml:3]: It is worth noting that the rule is not selected in any of our shipped RHEL 9 profiles as of b4612471aae69dd19eefed987038f1dbcd758237 . How about putting a comment to the rule.yaml file explaining why the rhel9 profile got there?

Comment (marcusburghardt) [reply linux_os/guide/system/auditing/package_audit-audispd-plugins_installed/rule.yml]: Managed by b95070ca34d3fb44a8f91faf8f3aae2257485954 , 2f2e088ec6d9088fb532a88c13ef71b9b40cd8b8 and 2d4af5f493832a2df29fb633db11bed5a428a87c

**Comment by marcusburghardt** [comment]
I believe we are good now. There is one issue that can be investigated in a separate PR: https://github.com/ComplianceAsCode/content/issues/11285

**Review by Mab879 — APPROVED** [review]
LGTM. 


Thank you for your hard work.

**Pull request merged**